File: app/src/main/java/com/example/tetrisapp/model/game/MockPlayfield.java
package com.example.tetrisapp.model.game;

import com.example.tetrisapp.interfaces.PlayfieldInterface;

public class MockPlayfield implements PlayfieldInterface {
    private String[][] state = new String[22][10];

    @Override
    public String[][] getState() {
        return state;
    }

    @Override
    public void setState(String[][] state) {
        this.state = state;
    }
}

File: app/src/main/java/com/example/tetrisapp/model/game/MockPlayfield.kt
package com.example.tetrisapp.model.game

import com.example.tetrisapp.interfaces.PlayfieldInterface

class MockPlayfield : PlayfieldInterface {
    override var state = Array(22) { arrayOfNulls<String>(10) }
    override fun isValidMove(matrix: Array<ByteArray>, rowOffset: Int, colOffset: Int): Boolean {
        return true
    }
}